arex-schedule-web-api/src/main/java/com/arextest/schedule/comparer/impl/DefaultReplayResultComparer.java

```diff
@@ -97,6 +102,15 @@ public boolean compare(ReplayActionCaseItem caseItem, boolean useReplayId) {
       MDCTracer.addPlanId(planId);
       MDCTracer.addPlanItemId(caseItem.getPlanItemId());
 
+      if (invalidReplayCaseService.isInvalidCase(caseItem.getTargetResultId())) {
+        caseItemRepository.updateCompareStatus(caseItem.getId(),
+                CompareProcessStatusType.ERROR.getValue());
+        caseItem.setCompareStatus(CompareProcessStatusType.ERROR.getValue());
+        comparisonOutputWriter.writeIncomparable(caseItem,
+                CaseSendStatusType.REPLAY_SERVICE_EXCEPTION.name());
```

Comment:
It is recommended to use a clearer naming to indicate the cause of this failure.